reader.go

```diff
@@ -1422,6 +1427,29 @@ func (r *Reader) ReadLag(ctx context.Context) (lag int64, err error) {
 	return
 }
 
+func (r *Reader) ConsumerGroupOffsets() (map[int]int64, error) {
```

Comment:
```suggestion
func (r *Reader) GroupOffsets() (map[int]int64, error) {
```
We don't use the term `ConsumerGroup` in the API currently, but we do have `GroupID` in the reader configuration, this could keep the naming a bit more consistent.